UOP1_Project/Assets/Scripts/Characters/Character.cs

```diff
@@ -23,6 +23,9 @@ public class Character : MonoBehaviour
     private float verticalMovement = 0f; //Represents how much a player will move vertically in a frame. Affected by gravity * gravityContributionMultiplier
     private Vector3 inputVector; //Initial input horizontal movement (y == 0f)
     private Vector3 movementVector; //Final movement vector
+    private bool isAboveSlopeLimit = false; // Used to check if current ground is above Slope Limit set on Character Controller
+    public bool isSliding = false; // Detects if character is currently sliding down a slope
```

Comment:
there's no point of this being public, should just be private

Reply:
Yeah, I changed it to public in order to see it better on the editor while developing and forgot to change it back

Reply:
you an also do 
[SerializedField]
private float blabla

and it'll show up on inspector, just a fyi .